TODO

```diff
@@ -17,3 +17,5 @@ The tasks are listed in a totally random order:
 . Import the text of all other translations
 
 . Implement user-defined annotations (major task, can wait until version 2)
+
+. Make all text simultaneously scrolling by select item in TOC and by clicking on search result
```

Comment:
Please add the word "optionally" before "simultaneously" and I'll merge your PR. Explanation: making all text columns scroll in sync unconditionally is not always convenient. Imagine the following (quite common) situation: you are using text column 1 for searching and wish to keep the text in column 2 (and maybe 3 as well) intact. For example, you may be following the reading in a study group from column 2, but simultaneously searching something in column 1. Synchronous scrolling would spoil this setup by losing the place in column 2. And you can always manually click on the text id in column 1 to sync columns 2 and 3 with it. However, I agree that _optionally_ it is a good feature to have, controlled via flag on the main panel.